docs/WalletGuide.md

```diff
@@ -699,3 +707,224 @@ suspend fun main() {
+
+You can read more about SEP-30 identities [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0030.md#common-request-fields)
+
+Next, let's create a recoverable account:
+```kotlin
+  val recoverableWallet =
+    recovery.createRecoverableWallet(
+      RecoverableWalletConfig(
+        account,
+        deviceKey,
+        AccountThreshold(10, 10, 10),
+        mapOf(first to identities, second to identities),
+        SignerWeight(10, 5),
+        sponsor
+      )
+    )
+```
+With given parameters, this function will:
+1. Create `account` on the Stellar network (if it's not created yet).
+2. `sponsor` will be used to sponsor all operations. You may not sponsor creating recoverable wallet, but in that case `account` must already be created.
+3. `deviceKey` will be used as a primary account key. Please note that master key belonging to `account` will be locked. `deviceKey` should be used as a primary signer instead.
+4. Set all account thresholds to 10. You can read more about threshold in the [documentation](https://developers.stellar.org/docs/encyclopedia/signatures-multisig#thresholds)
+5. Use identities that were defined earlier on both servers. (That means, both server will accept SEP-10 authentication via `recoveryKey` as an auth method)
+6. Set device key weight to 10, and recovery server weight to 5. Given account thresholds, both servers must be used to recover account, as transaction signed by one will only have weight of 5, which is not sufficient to change account key.
+
+Finally, sign and submit transaction to the network
+```kotlin
+  val tx = recoverableWallet.transaction.sign(account).sign(sponsor)
+
+  wallet.stellar().submitTransaction(tx)
+
+  val recoverySigners = recoverableWallet.signers
+```
+Note, that sponsor must sign key, as well as master key of the account. (After transaction is executed, you won't be able to use master key anymore)
+
+After transaction is executed, new account will be created.
+1. Address of account will be `account.address`
+2. `account` key will not be usable. Transaction signed with this key won't have enough authority.
+3. Primary signer with weight 10 will be `deviceKey` key.
+4. Recovery signers with weight 5 will be stored in `recoverySigners` list.
+5. All reserves for this account are sponsored by `sponsor`.
+6. Account will have 0 XLM on its balance.
+
+> You can get the full code [here](../examples/documentation/src/example-recovery-02.kt).
+
+### Get account info
+
+<!--- INCLUDE
+val recoveryKey = SigningKeyPair.fromSecret("<recovery key>")
+val account = "<account address>".toPublicKeyPair()
+
+suspend fun accountInfo() {
+-->
+
+<!--- SUFFIX 
+}
+-->
+
+You can fetch account info from one or more servers. To do so, first we need to authenticate,
+```kotlin
+  val auth1 = recovery.sep10Auth(first).authenticate(recoveryKey)
+  val auth2 = recovery.sep10Auth(second).authenticate(recoveryKey)
```

Comment:
I don't understand where `first` and `second` come from.

Reply:
I think it's defined earlier in the document